charts/timed/templates/configmap-backend.yaml

```diff
@@ -25,7 +25,7 @@ data:
   DJANGO_OIDC_LASTNAME_CLAIM: {{ .Values.auth.oidc.claims.lastname | quote }}
 {{- if .Values.auth.oidc.introspect.enabled }}
   DJANGO_OIDC_CHECK_INTROSPECT: "True"
-  DJANGO_OIDC_INTROSPECT_ENDPOINT: {{ .Values.auth.oidc.introspect.endpoint | quote }}
+  DJANGO_OIDC_INTROSPECT_ENDPOINT: {{ printf "%s/token/introspect" .Values.auth.oidc.url | quote }}
```

Comment:
This is specific to our using it with keycloak and we'll need to change it if we end up using a different IdP.